src/components/Avatar/AvatarStack/AvatarStack.style.ts

```diff
@@ -1,20 +1,22 @@
 import { css, SerializedStyles } from '@emotion/react';
+import { flex } from 'theme/functions';
 import { rem } from 'theme/utils';
 
 import { AvatarSizes } from '../Avatar';
+import { sizeBasedOnProp } from '../Avatar.style';
 
-const iconMarginLeft = (size: AvatarSizes) => {
-  switch (size) {
-    case 'lg':
-      return rem(-8);
-    case 'xs':
-      return rem(-2);
-    default:
-      return rem(-4);
-  }
-};
+const OVERLAP_FACTOR = 0.8;
```

Comment:
has this been discussed with the design team?

Reply:
yeap